spring-integration-file/src/main/java/org/springframework/integration/file/tail/FileTailingMessageProducerSupport.java

```diff
@@ -118,6 +136,78 @@ protected void publish(String message) {
 		}
 	}
 
+
+
+	@Override
+	protected void doStart() {
+		super.doStart();
+		if (this.idleEventInterval > 0) {
+			this.idleEventScheduledFuture = getTaskScheduler().scheduleWithFixedDelay(() -> {
+				long now = System.currentTimeMillis();
+				long lastAlertAt = FileTailingMessageProducerSupport.this.lastNoMessageAlert.get();
```

Comment:
Having this `Runnable` as a Lambda we don't need class prefix for properties any more. 😄